facilitator/src/task.rs

```diff
@@ -66,6 +66,8 @@ pub trait Task: Debug + Display + Sized + serde::de::DeserializeOwned {}
 #[derive(Debug, Deserialize, PartialEq)]
 #[serde(rename_all = "kebab-case")]
 pub struct IntakeBatchTask {
+    /// The trace identifier for the intake, typically a UUID
+    pub trace_id: Option<String>,
```

Comment:
If the trace IDs are UUIDs, let's enforce that by making this `Option<Uuid>` so that we can error out if we get malformed trace IDs.